dal/db.go

```diff
@@ -64,15 +66,15 @@ func (db DB) InitDB() error {
 
 // InsertTx inserts a new transaction into the database
 func (db DB) InsertTx(tx Tx) error {
```

Comment:
Should we use *DB, because we modify the db data?

Reply:
You are right. Updated it